models/biocro/tests/testthat/test.met2model.R

```diff
@@ -0,0 +1,22 @@
+context("met2model")
+
+outfolder <- tempfile()
+setup(dir.create(outfolder, showWarnings = FALSE))
+teardown(unlink(outfolder, recursive = TRUE))
+
+test_that("Met conversion runs without error", {
+  skip("BIOCRO met2model is currently broken!")
```

Comment:
Here's the error. Looks like the time zone parsing issue:

```
test.met2model.R:15: error: Met conversion runs without error
run end date 2000-12-31 23:59:59 after met data ends 2000-12-31 21:00:00
1: met2model.BIOCRO(in.path, in.prefix, outfolder, lat = 45.25, lon = -84.75, start_date = start_date, 
       end_date = end_date) at /Users/shik544/Box Sync/Projects/pecan_project/pecan/models/biocro/tests/testthat/test.met2model.R:15
2: PEcAn.data.atmosphere::load.cfmet(met.nc, lat = lat, lon = lon, start.date = yrstart, 
       end.date = yrend) at /Users/shik544/Box Sync/Projects/pecan_project/pecan/models/biocro/R/met2model.BIOCRO.R:71
3: PEcAn.logger::logger.severe("run end date", end.date, "after met data ends", max(all.dates$date))
4: stop(paste(msg, ...))
```

Reply:
Oh, right. I remember running into this and thinking it might be related to #1932, but had assumed it was fixed by now. Weird that it's only affecting BioCro, though.

Maybe open a placeholder issue and skip with a reference to it?

Reply:
To be fair, I think it's a problem in `load.cfmet`, not `met2model.BIOCRO`, but the latter is one of the relatively few `met2model` functions that uses `load.cfmet`. But will do.

Reply:
Opened #2274. Addressed in https://github.com/PecanProject/pecan/pull/2273/commits/361872fb46af73b5a2b10fb46cb7922f36dd7571.